emcc.py

```diff
@@ -1144,13 +1144,8 @@ def check(input_file):
         forced_stdlibs += ['libcxxabi']
 
       if not shared.Settings.ONLY_MY_CODE:
-        if type(shared.Settings.EXPORTED_FUNCTIONS) in (list, tuple):
-          # always need malloc and free to be kept alive and exported, for internal use and other modules
-          for required_export in ['_malloc', '_free']:
-            if required_export not in shared.Settings.EXPORTED_FUNCTIONS:
-              shared.Settings.EXPORTED_FUNCTIONS.append(required_export)
-        else:
-          logging.debug('using response file for EXPORTED_FUNCTIONS, make sure it includes _malloc and _free')
+        # always need malloc and free to be kept alive and exported, for internal use and other modules
+        shared.Settings.EXPORTED_FUNCTIONS += ['_malloc', '_free']
```

Comment:
If EXPORTED_FUNCTIONS already had _malloc and _free in it, does anything weird happen if they're in there twice?

Reply:
No.  Perhaps we should dedup the whole list at some point I guess?  But right here isn't the right place for it.

Reply:
It's weird because this patch adds a dedup check in system_libs.py, but removes one here.

Crazy idea: change EXPORTED_FUNCTIONS from a `list` to a `set`, and never worry about duplication again?

Reply:
Fixed this more completely.